i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md

```diff
@@ -0,0 +1,87 @@
+    style={{width:"100%", height:"500px", border:"0", borderRadius:"4px", overflow:"hidden"}}
+    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+></iframe>
+
+<br/>Notez que vous pouvez souvent combiner ces deux étapes en une seule. Vous pouvez demander à l'IA de générer directement un e-mail de réponse à partir de l'e-mail que vous avez reçu.
+
+## Emails Froids
+
+Les emails froids sont des emails envoyés à des personnes que vous ne connaissez pas. Il est difficile d'obtenir une réponse à partir de ces emails, donc il peut être utile d'envoyer beaucoup d'e-mails personnalisés. Voyons comment faire cela avec GPT-3(@bonta2022how).
+
+<div trydyno-embed="" openai-model="text-davinci-003" initial-prompt="Company name: Nightfall\nMy name: Max\nWrite a cold outreach email to this founder, named Pranav, pitching him our product, which is a neo-CRM. Make the email formal, yet approachable." initial-response="Subject: A modern CRM to help your business grow\n\nDear Pranav,\n\nMy name is Max and I'm the founder of Nightfall, a neo-CRM that helps businesses streamline their customer relationship management.\n\nWe understand that managing customer relationships can be a challenge, especially as your business grows. That's why we created Nightfall, a modern CRM that helps you stay organized and connected with your customers.\n\nNightfall offers a range of features, including automated customer segmentation, personalized customer journeys, and real-time analytics. We also provide 24/7 customer support to ensure that you get the most out of our product.\n\nI'd love to discuss how Nightfall can help your business grow. Would you be available for a call sometime this week?\n\nI look forward to hearing from you.\n\nBest,\nMax" max-tokens="256" box-rows="10" model-temp="0.0" top-p="0">
```

Comment:
will continue translating later to avoid a stale branch

Reply:
sounds good, ty